### PR TITLE
docs: add example and instructions for BYO ECR repos

### PR DESCRIPTION
**Pull request by deeno35**
Most enterprise customers will want this after they get to first base.

```
$ ./list_bigeye_images.sh
{
  "version": "0.0.0-brian",
  "imagePushedAt": "2023-11-08T14:43:39-08:00"
}
{
  "version": "0.0.0-brian-01",
  "imagePushedAt": "2023-11-09T06:17:20-08:00"
}
{
  "version": "0.0.0-brian-02",
  "imagePushedAt": "2023-11-09T15:14:05-08:00"
}
{
  "version": "0.0.0-brian-03",
  "imagePushedAt": "2023-12-05T09:20:31-08:00"
}
{
  "version": "1.32.0-hotfix.1",
  "imagePushedAt": "2023-12-13T12:48:53-08:00"
}
{
  "version": "1.32.0-hotfix.2",
  "imagePushedAt": "2023-12-13T20:35:01-08:00"
}
```

### CODE REVIEW

File: examples/minimal/main.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module "bigeye" {
```
Comment (deeno35) [inline examples/minimal/main.tf:1]: This is just cut/paste.  It used to be the "standard" install, but has been renamed as the "minimal" install.

File: examples/self-managed-ecr/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+See the "Standard" example for more details on general Bigeye stack configuration
+
+## Prerequisites
+
+Contact Bigeye support with your AWS account ID for access to Bigeye docker images.
+
+## Steps
+
+1. Use the Terraform example example in this repo to create local ECR repos in your AWS account.
```
Comment (BrianKopp) [inline examples/self-managed-ecr/README.md:21]: Typo? Double example

File: examples/standard/main.tf [hunk 3]
```diff
@@ -3,10 +3,27 @@ module "bigeye" {
   environment = "test"
   instance    = "bigeye"
 
-  # Your parent DNS name here, e.g. bigeye.my-company.com
-  top_level_dns_name = ""
+  # This is domain name of the domain that you have already set up in route53.  This terraform will create DNS entries
```
Comment (BrianKopp) [inline examples/standard/main.tf:6]: Maybe "hosted zone that you already have set up"? The language that R53 uses

File: examples/self-managed-ecr/main.tf [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  # Your parent DNS name here, e.g. bigeye.my-company.com
+  top_level_dns_name = ""
+
+  image_registry = format("%s.dkr.ecr.us-west-2.amazonaws.com", aws_ecr_repository.bigeye_ecr[0].registry_id)
+  # Get this from Bigeye Sales
+  image_tag = ""
+}
+
+# Create local ECR repos for caching of images.  See run list_bigeye_images.sh and get_bigeye_image.sh for
```
Comment (BrianKopp) [inline examples/self-managed-ecr/main.tf:14]: Is "see" meant to be here?

**Review by BrianKopp — APPROVED** [review]

**Pull request merged**